applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -63,20 +63,13 @@ BUILD_ASSERT(ARRAY_SIZE(headsets) >= CONFIG_BT_AUDIO_UNICAST_CLIENT_ASE_SRC_COUN
 	     "We need to have at least one headset device per ASE SOURCE");
 
 static le_audio_receive_cb receive_cb;
-static struct bt_audio_stream audio_streams[CONFIG_BT_AUDIO_UNICAST_CLIENT_ASE_SNK_COUNT +
-					    CONFIG_BT_AUDIO_UNICAST_CLIENT_ASE_SRC_COUNT];
+
 static struct bt_audio_unicast_group *unicast_group;
 static struct bt_codec *remote_codec[CONFIG_BT_AUDIO_UNICAST_CLIENT_PAC_COUNT * CONFIG_BT_MAX_CONN];
 
-static struct bt_audio_lc3_preset lc3_preset_nrf5340 = BT_AUDIO_LC3_UNICAST_PRESET_NRF5340_AUDIO;
-
-struct worker_data {
-	uint8_t stream_index;
-	uint8_t retries;
-} __aligned(4);
-K_MSGQ_DEFINE(kwork_msgq, sizeof(struct worker_data), ARRAY_SIZE(audio_streams), 4);
-
-static struct k_work_delayable stream_start_work[ARRAY_SIZE(audio_streams)];
```

Comment:
I could remember the reason we have this retry mechanism could be
1. There was a chance that we try to connect two CISes in parallel but not in sequential (we should wait first connected then start the second)
2. The CIS connection could fail in high stress condition
If we remove this part, do we see some misbehavior happens?

Reply:
I have tried to provoke errors for starting the stream, but I have not seen any issues.